README.md

```diff
@@ -224,10 +224,122 @@ __Objective-C__
     }];
 ```
 
-If you are using the IDFA to track your user, please beware that since iOS 14 Apple is more strict on the IDFA use cases. Please check Apple [documentation on which conditions](https://developer.apple.com/app-store/user-privacy-and-data-use/) and [how to retrieve the user's IDFA](https://developer.apple.com/documentation/apptrackingtransparency?language=objc).
+# iOS 14 and collecting IDs
+
+Due to changes introduced in iOS 14, VelocidiSDK no longer uses the IDFA by default and requires the developer to explicit an ID to identify the user. It is up to the developer to choose which user IDs to use. As a rule of thumb, we recommend reading Apple's
+instructions on [User Privacy and Data Use](https://developer.apple.com/app-store/user-privacy-and-data-use/).
+
+## Using your own first-party ID
+You can use any first-party ID with [VSDKUserId](https://ios.developers.velocidi.com/Classes/VSDKUtil.html), as exemplified in
+[Make a Match](#Make-a-match) and [Send a tracking event](#Send-a-tracking-event).
+
+## Using the IDFV
+The [Identifier for Vendor (IDFV)](https://developer.apple.com/documentation/uikit/uidevice/1620059-identifierforvendor) is an ID that is shared amongst apps from
+the same vendor in the same device. It allows a vendor to uniquely identify a user's device, without relying on the device-wide IDFA, and its respective limitations.
+
+__Swift__
+```swift
+let idfv = UIDevice.current.identifierForVendor!.uuidString
+let userId = VSDKUserId(id: idfv, type: "idfv")
+```
+
+__Objective-C__
+```objectivec
+NSString *idfv = [[[UIDevice currentDevice] identifierForVendor] UUIDString];
+VSDKUserId *userId = [[VSDKUserId alloc] initWithId:idfv type:@"idfv"];
+```
+
+## Using the IDFA
+If you are using the Identifier for Advertisers (IDFA) to identify the user, please make sure to read Apple's instructions on
+[User Privacy and Data Use](https://developer.apple.com/app-store/user-privacy-and-data-use/) and be sure that your use case is compliant with Apple's guidelines.
+
+Using the IDFA requires explicit authorization from the user for each application. To do so, start by establishing the message that will be presented to the user in the permission dialog. This can be done by adding the __NSUserTrackingUsageDescription__ key to the Info.plist file of the application. The value of this key will be the message presented.
+
+![tracking usage description entry](./docs/ios14_prompt_message.png)
+
+Before starting to send events, we have to request permission to access the IDFA, and only if that permission is granted, can we send events with
```

Comment:
I think we should more stronly present the following code excerpts as examples, stating that it is the developers decision when, where and how to present the permission dialog to the user, and the following are just utility methods that can be used if one does not care to control the moment when a permission dialog is shown to the user...

Reply:
Done in ce9fb2d